src/components/shell/timeIntervalDropdown/timeIntervalDropdown.container.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
```

Comment:
Is translate (please change to withNamespaces) needed here? If so, did it already rely on t being passed as a prop? Have those places been cleaned up now that the container supplies it?